### DISCUSSION TITLE
Farming without water?

### DISCUSSION THREAD

**Discussion by gamegam** [community/community#56687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is it possible for Pocketmine-MP not to change to soil if there is water, but not to change to soil even if there is it possible?

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/pmmp/PocketMine-MP? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
